frontend/fingertips-frontend/components/forms/IndicatorSort/indicatorSort.types.ts

```diff
@@ -0,0 +1,11 @@
+export enum SortOrderKeys {
+  relevance = 'relevance',
+  updated = 'updated',
+  alphabetical = 'alphabetical',
+}
+
+export const sortOrderLabels: Record<SortOrderKeys, string> = {
+  relevance: 'Most relevant',
+  updated: 'Last updated',
+  alphabetical: 'Alphabetical',
```

Comment:
Think you can do the following here;
```
{
  [SortOrderKeys.relevance]: 'Most relevant',
  [SortOrderKeys.update]: 'Last updated',
  [SortOrderKeys.alphabetical]: 'Alphabetical'
}
```

This should simplify in IndicatorSort to
```
<option key={key} value={key}>
   {sortOrderLabels[key]}
</option>
```